packaging/Dockerfile.fedora-nmstate-dev

```diff
@@ -44,10 +45,9 @@ RUN dnf -y install --setopt=install_weak_deps=False \
                    # Below package for pip (used by tox) to build dbus-python
                    make \
                    dbus-devel && \
-    alternatives --install /usr/bin/python python /usr/bin/python3 1 && \
-    ln -s /usr/bin/pytest-3 /usr/bin/pytest && \
     dnf clean all && \
-    pip3 install --upgrade pip pytest==4.6.6 pytest-cov==2.8.1 && \
+    pip install pytest==4.6.6 pytest-cov==2.8.1 --user && \
+    ln -s /root/.local/bin/pytest /usr/bin/pytest && \
```

Comment:
IMHO this should be fixed by copying `/etc/skel/.bashrc` to `/root/.bashrc`

Reply:
FIled a PR in Fedora to get this fixed: https://src.fedoraproject.org/rpms/rootfiles/pull-request/1

Reply:
By using `cp /etc/skel/.bashrc /root/.bashrc`, I need to explain it.
By using `ln -s /root/.local/bin/pytest /usr/bin/pytest`, people could know I just point the pip installed pytest to PATH.

Once fedora fix this problem, we don't need it at all. So I like to use the `ln` way.

Reply:
Can you add a comment that it can be removed with Fedora 32?